media/evolutionary_biologists.txt

```diff
@@ -0,0 +1,392 @@
+A
```

Comment:
I reckon we should remove the A, B, C etc headers and also the parentheses?

Reply:
Roger. Have kept newlines between the surnames with different starting capital letters.